data-types.md

```diff
@@ -118,7 +114,7 @@ A standard point of vehicle telemetry. References to latitude and longitude impl
 | `telemetry_id`    | UUID            | Required               | ID used for uniquely-identifying a Telemetry entry |
 | `timestamp`       | [Timestamp][ts] | Required               | Date/time that event occurred. Based on GPS or GNSS clock            |
 | `trip_ids`        | UUID[]          | Required               | If telemetry occurred during a trip, the ID of the trip(s).  If not in a trip, `null`. 
-| `journey_id`      | UUID            | Required               | If telemetry occurred during a trip, the ID of the journey.  If not in a trip, `null`.
+| `journey_ids`      | UUID[]            | Required               | If telemetry occurred during a trip, the ID of the journey(s).  If not in a journey, `null`.
```

Comment:
I thought that by definition there could only be one journey at a time?

Reply:
Ah I see, yes. I was thinking somehow if it spanned multiple modes or something. You are right, should remain singular.